### PR TITLE
[Polkadot Wiki Migration] Validator Payout

### PR DESCRIPTION
**Pull request by CrackTheCode016**
> This page was migrated from https://wiki.polkadot.network/docs/maintain-guides-validator-payout

### CODE REVIEW

**Comment by dawnkelly09** [comment]
All I did was merge master into this to resolve merge conflicts but it triggered a content team review I can't cancel. Sorry about that!

File: infrastructure/validators/rewards-payout.md [hunk 1]
```diff
@@ -0,0 +1,216 @@
...
+
+For every era (a period of time approximately 6 hours in length in Kusama, and 24 hours in Polkadot), validators are paid proportionally to the amount of [era points](TODO: path) they have collected. Era points are reward points earned for payable actions like:
+
+- Issuing validity statements for [parachain](TODO: path) blocks
+- Producing a non-uncle block in the relay chain
+- Producing a reference to a previously unreferenced uncle block
+- Producing a referenced uncle block
+
+!!!note
+    An uncle block is a Relay Chain block that is valid in every regard, but which failed to become canonical. This can happen when two or more validators are block producers in a single slot, and the block produced by one validator reaches the next block producer before the others. We call the lagging blocks uncle blocks.
```
Comment (nhussein11) [inline infrastructure/validators/rewards-payout.md:21]:
```suggestion
    An uncle block is a Relay Chain block that is valid in every regard but has failed to become canonical. This can happen when two or more validators are block producers in a single slot, and the block produced by one validator reaches the next block producer before the others. The lagging blocks are called uncle blocks.
```

File: infrastructure/validators/rewards-payout.md [hunk 2]
```diff
@@ -0,0 +1,216 @@
...
+
+Payments occur at the end of every era.
+
+Era points create a probabilistic component for staking rewards.
+
+If the mean of staking rewards is the average rewards per era, then the variance is the variability from the average staking rewards. The exact DOT value of each era point is not known in advance since it depends on the total number of points earned by all validators in a given era. This is designed this way so that the total payout per era depends on Polkadot's [inflation model](), and not on the number of payable actions (for example, authoring a new block) executed. For more information, read this StackExchange post about [reward calculations](https://substrate.stackexchange.com/questions/5353/how-are-rewards-in-dot-calculated-from-the-era-points-earned-by-validators-in-po).
```
Comment (nhussein11) [inline infrastructure/validators/rewards-payout.md:27]: I'm wondering here if we need to refer to DOT exactly or if this should be more generic and say something like: "the relay chain native token", covering both DOT and KSM

Comment (nhussein11) [inline infrastructure/validators/rewards-payout.md:27]:
```suggestion
If the mean of staking rewards is the average rewards per era, then the variance is the variability from the average staking rewards. The exact DOT value of each era point is not known in advance since it depends on the total number of points earned by all validators in a given era. This is designed this way so that the total payout per era depends on Polkadot's [inflation model](), and not on the number of payable actions (for example, authoring a new block) executed. For more information, read the [How are rewards in dot calculated from the era point](https://substrate.stackexchange.com/questions/5353/how-are-rewards-in-dot-calculated-from-the-era-points-earned-by-validators-in-po) post.
```

File: infrastructure/validators/rewards-payout.md [hunk 3]
```diff
@@ -0,0 +1,216 @@
...
+
+With parachains now on Polkadot, a large percentage of era points will come from parachain validation, as a subset of validators are selected to para-validate for all parachains each epoch, and those para-validators can generate more era points as a result. Para-validators are rewarded 20 era points each for each parachain block that they validate.
```
Comment (nhussein11) [inline infrastructure/validators/rewards-payout.md:29]: Probably this was written when parachains went live. I'd paraphrase this a bit since they have been a reality for some time now

File: infrastructure/validators/rewards-payout.md [hunk 4]
```diff
@@ -0,0 +1,216 @@
...
+
+    - `v` = the variance of staking rewards,
+    - `p` = number of para-validators,
+    - `w` = number validators in the active set,
+    - `e` = era,
+
+    Then, `v` &#8593; if `w` &#8593;, as this reduces `p` : `w`, with respect to `e`.
+
+    Increased `v` is expected, and initially keeping `p` &#8595; using the same para-validator set
+    for all parachains ensures [availability]() and [voting](../learn/learn-polkadot-opengov.md). In addition, despite `v` &#8593; on an `e` to `e` basis, over time, the amount of rewards each validator receives will equal out based on the continuous selection of para-validators.
```
Comment (nhussein11) [inline infrastructure/validators/rewards-payout.md:59]: IMO it would be better for readability to also explicitly write down these relationships, not just use arrows and icons. However, I do like the arrows, so I'd keep both since I believe they are easier to remember

File: infrastructure/validators/rewards-payout.md [hunk 5]
```diff
@@ -0,0 +1,216 @@
...
+
+!!!info
+    Validators set their preference as a percentage of the block reward, _not_ an absolute number of
+    DOT. Polkadot's block reward is [based on the _total_ amount at stake](../learn/learn-inflation.md#ideal-staking-rate). The
+    commission is set as the amount taken by the validator; that is, 0% commission means that the
+    validator does not receive any proportion of the rewards besides that owed to it from self-stake,
+    and 100% commission means that the validator operator gets all rewards and gives none to its
+    nominators.
+
+In the following examples, we can see the results of several different validator payment schemes and
```
Comment (nhussein11) [inline infrastructure/validators/rewards-payout.md:160]: As you are using DOT in these examples, I'd add a small note or sentence that explains that these example are thought for polkadot relay chain that uses DOT as native token

File: infrastructure/validators/rewards-payout.md [hunk 6]
```diff
@@ -0,0 +1,216 @@
...
+nominator may stake different validators.
+
+Each validator in the example has selected a different validator payment (that is, a percentage of
+the reward set aside directly for the validator before sharing with all bonded stake). The
+validator's payment percentage (in DOT, although the same calculations work for KSM) is listed in
+brackets (`[]`) next to each validator. Note that since the validator payment is public knowledge,
+having a low or non-existent validator payment may attract more stake from nominators, since they
+know they will receive a larger reward.
+
+```
```
Comment (nhussein11) [inline infrastructure/validators/rewards-payout.md:174]: I'm still unsure if this would look nicer in a table format...

**Comment by dawnkelly09** [comment]
Made some pretty heavy edits trying to make this a bit more approachable. @nhussein11 please make sure I'm not leading anyone astray technically. @eshaben please take a look at the mermaid diagrams and let me know wyt.

File: infrastructure/validators/rewards-payout.md [hunk 7]
```diff
@@ -0,0 +1,225 @@
...
+
+To illustrate this relationship:
+
+- Para-validator era points tend to have a higher impact on the expected value of staking rewards compared to non-para-validator points
+- The variance in staking rewards increases as the total number of validators grows relative to the number of para-validators
+- In simpler terms, when more validators are added to the active set without increasing the para-validator pool, the disparity in rewards between validators becomes more pronounced
+
+However, despite this increased variance, rewards tend to even out over time due to the continuous rotation of para-validators across eras. The network's design ensures that over multiple eras, each validator has an equal opportunity to participate in para-validation, eventually leading to a balanced distribution of rewards.
+
+??? function "Probability in Staking Rewards"
```
Comment (eshaben) [inline infrastructure/validators/rewards-payout.md:37]: I think either a `note` or `interface` admonition might look a little bit better in this case

File: infrastructure/validators/rewards-payout.md [hunk 8]
```diff
@@ -0,0 +1,225 @@
...
+
+    This should only serve as a high-level overview of the probabilistic nature for staking rewards.
+
+    Let:
+
+    - `pe` = para-validator era points,
+    - `ne` = non-para-validator era points,
+    - `EV` = expected value of staking rewards,
```
Comment (eshaben) [inline infrastructure/validators/rewards-payout.md:45]:
```suggestion
    - `pe` = para-validator era points
    - `ne` = non-para-validator era points
    - `EV` = expected value of staking rewards
```

File: infrastructure/validators/rewards-payout.md [hunk 9]
```diff
@@ -0,0 +1,225 @@
...
+
+    Since `EV(pe)` has a more weighted probability on the `EV`, the increase in variance against the `EV` becomes apparent between the different validator pools (aka. validators in the active set and the ones chosen to para-validate).
+
+    Also, let:
+
+    - `v` = the variance of staking rewards,
+    - `p` = number of para-validators,
+    - `w` = number validators in the active set,
+    - `e` = era,
+
```
Comment (eshaben) [inline infrastructure/validators/rewards-payout.md:57]:
```suggestion
    - `v` = the variance of staking rewards
    - `p` = number of para-validators
    - `w` = number validators in the active set
    - `e` = era

```

File: infrastructure/validators/rewards-payout.md [hunk 10]
```diff
@@ -0,0 +1,225 @@
...
+    Then, `v` &#8593; if `w` &#8593;, as this reduces `p` : `w`, with respect to `e`.
+
+    Increased `v` is expected, and initially keeping `p` &#8595; using the same para-validator set for all parachains ensures [availability](TODO: add path) and [voting](../learn/learn-polkadot-opengov.md). In addition, despite `v` &#8593; on an `e` to `e` basis, over time, the amount of rewards each validator receives will equal out based on the continuous selection of para-validators.
```
Comment (eshaben) [inline infrastructure/validators/rewards-payout.md:60]:
```suggestion
    Increased `v` is expected, and initially keeping `p` &#8595; using the same para-validator set for all parachains ensures [availability](TODO: add path) and [voting](https://wiki.polkadot.network/docs/learn-polkadot-opengov/){target=\_blank}. In addition, despite `v` &#8593; on an `e` to `e` basis, over time, the amount of rewards each validator receives will equal out based on the continuous selection of para-validators.
```

File: infrastructure/validators/rewards-payout.md [hunk 11]
```diff
@@ -0,0 +1,225 @@
...
+
+Validator rewards are distributed equally among all validators in the active set, regardless of the total stake behind each validator. However, individual payouts may differ based on the number of era points a validator has earned. Although factors like network connectivity can affect era points, well-performing validators should accumulate similar totals over time.
+
+Validators can also receive tips from users, which incentivize them to include certain transactions in their blocks. Validators retain 100% of these tips.
+
+Rewards are paid out in the network's native token (KSM for Kusama and DOT for Polkadot). 
+
+The following example illustrates a four member validator set with their names, amount they have staked, and how payout of rewards is divided. This scenario assumes all validators earned the same amount of era points and no one received tips: 
+
+``` mermaid
```
Comment (eshaben) [inline infrastructure/validators/rewards-payout.md:74]: Nice! These look great. Once we get the styling in for these content pages, we'll be able to re-evaluate if we need to make any styling updates here, but this looks great!

File: infrastructure/validators/rewards-payout.md [hunk 12]
```diff
@@ -0,0 +1,225 @@
...
+  end
+    space
+    E["Payout (8 DOT total)"]:1
+    E --"2 DOT"--> A
+    E --"2 DOT"--> B
+    E --"2 DOT"--> C
+    E --"2 DOT"--> D 
+```
+
+Note that this is different than most other Proof-of-Stake systems. As long as a validator is in the validator set, it will receive the same block reward as every other validator. Validator Alice, who had 18 tokens staked, received the same 2 token reward in this era as Dave who had only 7 tokens staked.
```
Comment (eshaben) [inline infrastructure/validators/rewards-payout.md:93]:
```suggestion
Note that this is different than most other Proof-of-Stake systems. As long as a validator is in the validator set, it will receive the same block reward as every other validator. Validator Alice, who had 18 DOT staked, received the same 2 DOT reward in this era as Dave, who had only 7 DOT staked.
```

Comment (eshaben) [reply infrastructure/validators/rewards-payout.md:93]: Just to match the diagram 🤷‍♀️

**Review by eshaben — COMMENTED** [review]
Great job with the diagrams! 🙌 

Noticed a few other minor things 🙂

**Comment by dawnkelly09** [comment]
Changes made. Ready for final pass from Erin

**Review by nhussein11 — APPROVED** [review]
LGTM!

File: infrastructure/validators/rewards-payout.md (hunk 7 above)
Comment (eshaben) [reply infrastructure/validators/rewards-payout.md]: Looks gooood! ✨

**Review by eshaben — APPROVED** [review]
LGTM! 🚀
